src/static/css/skip-nav.less

```diff
@@ -0,0 +1,24 @@
+#skip-nav {
+    padding: 10px;
+    border: 2px solid @green;
+    border-top: 0;
+    border-bottom-right-radius: 8px;
+    border-bottom-left-radius: 8px;
+    position: absolute;
+    top: -50px;
+    left: 10px;
+    color: @pacific;
+    background: transparent;
+    transition: top 1s ease, background .5s linear;
```

Comment:
Since you're using `transition`, you may wan to use `transform` instead of `top`. It animates better and it'll be supported in any browser that supports `transition`.